draft-ietf-anima-brski-prm.md

```diff
@@ -336,84 +320,84 @@ Solution examples based on existing technology are provided with the focus on ex
-|       |<------>|           |<----->| (PKI RA)  |  .
-|       |     .  |     LDevID|       |           |  .
-|       |     .  +-----------+       +-----+-----+  .
-|IDevID |     .                            |        .
-|       |     .         +------------------+-----+  .
-+-------+     .         | Key Infrastructure     |  .
-              .         | (e.g., PKI Certificate |  .
-              .         |       Authority)       |  .
-              .         +------------------------+  .
-              .......................................
-                       "Domain" components
+                         +---------------------------+
+    +---- Drop Ship -----| Vendor Service            |
+    |                    +---------------+-----------+
+    |                    | M anufacturer |           |
+    |                    | A uthorized   | Ownership |
+    |                    | S igning      | Tracker   |
+    |                    | A uthority    |           |
+    |                    +---------------+-----------+
+    |                                         ^
+    |                                         | BRSKI-
+    |                                         | MASA
+    |          ...............................|.........
+    V          .                              v        .
++--------+     .  +------------+        +-----------+  .
+|        |     .  |            |        |           |  .
+| Pledge | BRSKI- | Registrar- | BRSKI- | Domain    |  .
+|        |  PRM   | Agent      |  EST   | Registrar |  .
+|        |<------>|            |<------>| (PKI RA)  |  .
+|        |     .  |     LDevID |        |           |  .
+|        |     .  +------------+        +-----+-----+  .
+| IDevID |     .                              |        .
+|        |     .           +------------------+-----+  .
++--------+     .           | Key Infrastructure     |  .
+               .           | (e.g., PKI Certificate |  .
+               .           |        Authority)      |  .
+               .           +------------------------+  .
+               .........................................
+                         "Domain" Components
 ~~~~
-{: #uc2figure title='BRSKI-PRM Architecture overview using registrar-agent' artwork-align="left"}
+{: #uc2figure title='BRSKI-PRM architecture overview using registrar-agent' artwork-align="left"}
 
 The following list describes the components in a (customer) site domain:
 
 * Pledge: The pledge is expected to respond with the necessary data objects for bootstrapping to the registrar-agent.
   The protocol used between the pledge and the registrar-agent is assumed to be HTTP in the context of this document.
-  Other protocols may be used like CoAP, Bluetooth, or NFC, but are out of scope of this document.
-  A pledge acting as a server during bootstrapping leads to some differences to BRSKI:
+  Other protocols such as CoAP, NFC, or Bluetooth may be used, but are out of scope of this document.
```

Comment:
Is this a "may" or "MAY". In general very few normative assertions in this document.

Reply:
In this case it is "may" as we do not specify their use.